<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spectroscopy</class>
 <widget class="QWidget" name="Spectroscopy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1371</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectroscopy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="measTab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportButton">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="saveconfocalBox">
           <property name="text">
            <string>Save confocal position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="temp_label">
           <property name="text">
            <string>Detector temperature: </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="2">
          <widget class="QSpinBox" name="exposuresBox">
           <property name="prefix">
            <string>Exposures per acquisition: </string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="exposuretimeBox">
           <property name="prefix">
            <string>Exposure time: </string>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QDoubleSpinBox" name="centerBox">
           <property name="prefix">
            <string>Center wavelength: </string>
           </property>
           <property name="suffix">
            <string> nm</string>
           </property>
           <property name="minimum">
            <double>100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>2000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>600.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QComboBox" name="baseconfBox"/>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="acquisitionsBox">
           <property name="suffix">
            <string> acquisitions (0 for inf)</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fiTab">
      <attribute name="title">
       <string>Fit</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>saveconfocalBox</tabstop>
  <tabstop>baseconfBox</tabstop>
  <tabstop>acquisitionsBox</tabstop>
  <tabstop>centerBox</tabstop>
  <tabstop>exposuretimeBox</tabstop>
  <tabstop>exposuresBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
